README.md

```diff
@@ -31,22 +32,28 @@
 
 1. On top of the fields already available in AB3, the user will be able to input the following:
     * Student's matriculation number
-    * Vaccination status/preference
+    * Vaccination status
     * Faculty
     * Residence on campus (if applicable)
     * Medical history
-1. A filter function that will allow the user to display a specific group of students
-1. The user will be able to view all the detailed information of a student at a glance. Currently, only
-   the summarised information on the UI can be viewed.
-1. Create vaccination appointments and display them on the UI.
+1. The `tag` field inherited from AB3 will be removed. 
+1. A `filter` function that will allow the user to display a specific group of students, by 
+   vaccination status, faculty or school residence. 
+1. Modify the `find` and `delete` commands to identify students by matriculation number.
+1. Create vaccination appointments and display them on the UI sorted by date.
+1. Introduce the use of icons in the UI.
 
 #### v1.3
 
-1. Detect duplicate entries.
-1. Sort vaccination appointments and display only upcoming ones.
-1. Provide statistical insights regarding the vaccination of the student population, by faculty, residence or other
-   relevant parameters.
-1. Undo feature to quickly correct accidental mistakes.
+1. Provide statistical insights regarding the vaccination of the student population, by faculty, residence or across 
+   the entire NUS. 
+1. Edit the details of existing vaccination appointments
+1. Delete existing vaccination appointments
+1. View the number of vaccination appointments in the past 6 days and in the next 6 days. 
```

Comment:
Sorry, for this line could it be 'within the past week' and 'in the next week' instead of 'in the past 6 days'?

Reply:
Is it not 6 days though?